utils/src/deployer.rs

```diff
@@ -440,7 +474,7 @@ pub async fn is_proxy_contract(
 #[derive(Clone, Copy, Debug, PartialEq, Eq, ValueEnum)]
 pub enum ContractGroup {
     #[clap(name = "hotshot")]
-    HotShot,
+    HotShot, // TODO: confirm whether we keep HotShot in the contract group
```

Comment:
do we still want to deploy HotShot?

Reply:
We can get rid of it but I think that can be a separate change, just to keep this as small as possible